torch/quantization/__init__.py

```diff
@@ -3,14 +3,20 @@
 from .observer import *  # noqa: F401
 from .QConfig import *  # noqa: F401
 from .fake_quantize import *  # noqa: F401
+import torch
 
 def default_eval_fn(model, calib_data):
     r"""
     Default evaluation function takes a torch.utils.data.Dataset or a list of
     input Tensors and run the model on the dataset
     """
-    for data in calib_data:
-        model(data)
+    total, correct = 0, 0
+    for data, target in calib_data:
+        output = model(data)
+        _, predicted = torch.max(output, 1)
+        total += target.size(0)
+        correct += (predicted == target).sum().item()
+    return correct / total
```

Comment:
why do we have this as a default function? it feels very specialized to classification problem

Reply:
I can rename that to test_only.. we just need some function to run the tests.